project-base/tests/App/Acceptance/acceptance/PageObject/Front/LoginPage.php

```diff
@@ -4,19 +4,30 @@
 
 namespace Tests\App\Acceptance\acceptance\PageObject\Front;
 
+use Facebook\WebDriver\WebDriverBy;
 use Tests\App\Acceptance\acceptance\PageObject\AbstractPage;
 
 class LoginPage extends AbstractPage
 {
-    /**
-     * @param string $email
-     * @param string $password
-     */
-    public function login($email, $password)
+    protected const DEFAULT_USER_NAME = 'Jaromír Jágr';
+    protected const DEFAULT_USER_EMAIL = 'no-reply@shopsys.com';
+    protected const DEFAULT_USER_PASSWORD = 'user123';
+
+    public function login()
```

Comment:
what about keeping a possibility to pass email and password into this method?
I can imagine a scenario when I need a different user for certain test

Reply:
Yeah in such case you change this method, because it is in project-base. Right now there is no use for it so I do not see any reason to keep it.

Reply:
I see... Okay, I can live with that